tests/release/service/happy_path.go

```diff
@@ -38,39 +41,51 @@ var _ = framework.ReleaseServiceSuiteDescribe("Release service happy path", Labe
 
 	var releaseCR *releaseApi.Release
 
+	var testEnvironment = utils.GetEnv("TEST_ENVIRONMENT", releasecommon.UpstreamTestEnvironment)
+
 	BeforeAll(func() {
-		// Initialize the tests controllers
-		fw, err = framework.NewFramework(utils.GetGeneratedNamespace("happy-path"))
-		Expect(err).NotTo(HaveOccurred())
-		devNamespace = fw.UserNamespace
-		managedNamespace = utils.GetGeneratedNamespace("happy-path-managed")
-		_, err = fw.AsKubeAdmin.CommonController.CreateTestNamespace(managedNamespace)
+		if testEnvironment == releasecommon.DownstreamTestEnvironment {
+			fw, err = framework.NewFramework(utils.GetGeneratedNamespace(devNamespace))
+			Expect(err).NotTo(HaveOccurred())
+			devNamespace = fw.UserNamespace
+			managedNamespace = utils.GetGeneratedNamespace(managedNamespace)
```

Comment:
Below sentence need to be added  in this block?
`kubeAdminClient = fw.AsKubeAdmin`